assets/javascripts/modules/ajaxCallbacks.js

```diff
@@ -57,6 +57,54 @@ var ajaxCallbacks = {
       }
     }
   },
+  apiSubscribeResponse: {
+    callbacks: {
+      success: function(response, $element, data, helpers) {
+        var $onDisabled = $('<span class="toggle--on">On</span>'),
+          $offDisabled = $('<span class="toggle--disabled">Off</span>'),
+          $parent = $element.parent(),
+          $off = $element.next(),
```

Comment:
Isn't this a bit to specific? Relying on the fact that "off" button is right after "on"?